plugin.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/appleboy/drone-git-push/repo"
```

Comment:
I'm guessing that @erkolson is new to Go. 

@erkolson please note that the Go package corresponds to its version control repository path. It should therefore not be changed. I'm guessing perhaps you changed because you had difficulty compiling locally? Please note the local path of your Go project should match the package path. Please read https://golang.org/doc/code.html#ImportPaths for more details.